src/util.f90

```diff
@@ -820,7 +820,7 @@ subroutine generate_bunch_bigaussian_weighted( &
  real(dp), allocatable :: ppcb_positions(:,:)
 
  bch=weight
- sigma_cut=3.0
+ sigma_cut=4.0
```

Comment:
Just don't change it anymore upstream if you think this is the best choice and you don't want to move it into an input parameter